mlrun/utils/helpers.py

```diff
@@ -267,14 +267,22 @@ def validate_artifact_key_name(
     )
 
 
-def validate_inline_artifact_body_size(body: typing.Union[str, bytes, None]) -> None:
+def validate_artifact_body_size(body: typing.Union[str, bytes, None]) -> None:
+    """
+    Validates the size of the artifact body.
+
+    :param body: The artifact body, which can be a string, bytes, or None.
+
+    :raises mlrun.errors.MLRunBadRequestError: If the body exceeds the maximum allowed size.
+    """
     if body and len(body) > MYSQL_MEDIUMBLOB_SIZE_BYTES:
         raise mlrun.errors.MLRunBadRequestError(
             "The body of the artifact exceeds the maximum allowed size. "
             "Avoid embedding the artifact body. "
             "This increases the size of the project yaml file and could affect the project during loading and saving. "
-            "More information is available at"
-            "https://docs.mlrun.org/en/latest/projects/automate-project-git-source.html#setting-and-registering-the-project-artifacts"
+            "More information is available at "
```

Comment:
A few comments about the error message.
1. I don't see any extra information in the link. IMO it can be removed.
2. No need to say "please" and don't specify parameters name of other methods you don't know where you cam from (could be packagers).